lib/config/migration.ts

```diff
@@ -426,9 +437,14 @@ export function migrateConfig(
             schedules[i] = schedules[i].replace(' every day', '');
           }
           if (
-            /every (mon|tues|wednes|thurs|fri|satur|sun)day$/.test(schedules[i])
+            regEx(/every (mon|tues|wednes|thurs|fri|satur|sun)day$/).test(
```

Comment:
We should some of those regex to module scope const to improve memory usage and performance.

This should be done in a seperate PR

Reply:
I checked the `node-re2` docs and it says to use `RegExp` instead of `re2` for such cases and some other cases too.

Reply:
@RahulGautamSingh can you be more explicit about exactly which docs sentence you are referring to?

Reply:
Above link takes us to the limitations section, under it we have 3 sections:
1. Backreferences
2. Lookahead Assertions
3. Mismatched Behavior

Under mismatched behavior this example
`var pattern = '(?:(a)|(b)|(c))+';`
seems similar to the one @viceice pointed out.

We also have some regex where we have lookaheads & backreferences in use so I have been testing almost each updated regex manually to prevent any failing test cases.